assets/src/edit-story/components/dropTargets/karma/background.karma.js

```diff
@@ -455,7 +455,7 @@ describe('Background Drop-Target integration', () => {
 });
 
 function getMediaLibraryElementByIndex(fixture, index) {
-  return fixture.querySelectorAll('[data-testid=mediaElement]')[index];
+  return fixture.querySelectorAll('.mediaElement')[index];
```

Comment:
Why not using `[data-testid^="mediaElement"]`?

The `[attribute^=value]` selector matches every element whose attribute value begins with a specified value.

Reply:
Fixed